index.html

```diff
@@ -8,38 +8,116 @@
   <style>
     .header {
       text-align: center;
+     
     }
+
+    #font {
+      font-family: "Century Gothic";
```

Comment:
In addition to setting styles to classes (prefixed with ".") and ids (prefixed with "#"), we can set styles to an element of a given type. So for example, to make all of our H1 tags red, we could do `h1 { color: red }`. So instead of adding an id to the body element, this should instead be `body { font-family: ... }`.

Note that this font is not considered "web safe," as many users will not have it on their machines. One of the nice things about the `font-family` declaration is that you can list multiple fonts so that if the user doesn't have your top choice installed, it will fallback to another option. So you might replace this line with something like: `font-family: "Century Gothic", Arial, sans-serif`

But of course the risk of having a non-standard font listed first is that many users will be looking at a very different version of the page than you are.

Here's a list of web safe fonts/font combinations: http://www.w3schools.com/cssref/css_websafe_fonts.asp

Alternatively, you could use a web font, which doesn't require the font to be installed on the user's computer. The most used source of web fonts is [Google Fonts](https://www.google.com/fonts).